docs/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ ETH_GAS_LIMIT_KEEPER_JOB_TYPE # EVM.GasEstimator.LimitKeeperJobType
 
 ### Changed
 
+- Keypath now supports paths with any depth, instead of limiting it to 2
```

Comment:
Should this be in a new section for the next release?

Reply:
I mean that we need to start a new release section for this change, since this grouping is for 1.7.0 which hasn't been merged back in to develop yet.

Reply:
IMHO we should be updating the changelog on develop as soon as we cut a release branch in order to prevent this mixup from happening in the first place, because it is quite common.